libfwupd/fwupd-enums.h

```diff
@@ -101,6 +103,8 @@ typedef enum {
 #define FWUPD_DEVICE_FLAG_NEEDS_BOOTLOADER	(1u << 6)	/* Since: 0.7.3 */
 #define FWUPD_DEVICE_FLAG_REGISTERED		(1u << 7)	/* Since: 0.9.7 */
 #define FWUPD_DEVICE_FLAG_NEEDS_REBOOT		(1u << 8)	/* Since: 0.9.7 */
+#define FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG_USER	(1u << 9)	/* Since: 1.0.3 */
+#define FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG_AUTO	(1u << 10)	/* Since: 1.0.3 */
```

Comment:
If this PR still gets merged, these need to bump up to the current version (1.0.5 right now) in the comment.

Reply:
Right, I'm putting this on the back burner for a bit; higher priority issues.